priv/repo/migrations/20250830152819_create_cutoffs_table.exs

```diff
@@ -8,7 +8,7 @@ defmodule Dbservice.Repo.Migrations.CreateCutoffsTable do
       add :college_id, references(:college, on_delete: :nothing)
       add :degree, :string
       add :branch_id, references(:branch, on_delete: :nothing)
```

Comment:
should we have unique index on this

Reply:
No, a unique branch record can have multiple cutoff values